dashboard/src/modules/components/ProfileComponent/ClipboardCopy.jsx

```diff
@@ -8,11 +8,9 @@ const ClipboardCopy = ({ copyText }) => {
   const [isCopied, setIsCopied] = useState(false);
 
   const copyTextToClipboard = async (text) => {
-    if ("clipboard" in navigator) {
-      return await navigator.clipboard.writeText(text);
-    } else {
-      return document.execCommand("copy", true, text);
-    }
+    "clipboard" in navigator
+      ? await navigator.clipboard.writeText(text)
+      : document.execCommand("copy", true, text);
   };
```

Comment:
As long as you're fixing this for Dave anyway....  😉 

Since this is now an expression, we don't need to curl braces (`{}`), you can omit them or replace them with parentheses.

(With the code block (i.e., the braces), I'm surprised that the function returns a value without using an explicit `return` statement.)